### PR TITLE
fix istioctl x metrics

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**

there's no `istio_request_duration_seconds` metics in /stats/prometheus

fix `istioctl x mertics` not working

cc @douglas-reid ptal

### CODE REVIEW

File: istioctl/cmd/metrics.go [hunk 1]
```diff
@@ -213,25 +207,43 @@ func metrics(promAPI promv1.API, workload string, duration time.Duration) (workl
 	return sm, nil
 }
 
+func getLantecy(promAPI promv1.API, workloadName, workloadNamespace string, duration time.Duration, quantile float64) (time.Duration, error) {
```
Comment (douglas-reid) [inline istioctl/cmd/metrics.go:210]: typo: replace "Lantecy"  with "Latency".

File: istioctl/cmd/metrics.go [hunk 2]
```diff
@@ -213,25 +207,43 @@ func metrics(promAPI promv1.API, workload string, duration time.Duration) (workl
...
 			return 0, nil
 		}
+
 		return float64(v[0].Value), nil
 	default:
 		return 0, errors.New("bad metric value type returned for query")
 	}
 }
 
+func convertLetencyToDuration(val float64) time.Duration {
```
Comment (douglas-reid) [inline istioctl/cmd/metrics.go:243]: typo: should be `convertLatencyToDuration`

**Review by douglas-reid — COMMENTED** [review]
Thanks for tackling this!  Just a few typos to clean up.

Just for your awareness, there is someone that is working on refactoring all of this code to provide more utility, etc. Hopefully, we'll see lots of improvements here soon.

File: istioctl/cmd/metrics.go (hunk 2 above)
Comment (zirain) [reply istioctl/cmd/metrics.go]: fixed

File: istioctl/cmd/metrics.go (hunk 1 above)
Comment (zirain) [reply istioctl/cmd/metrics.go]: fix typo

**Review by douglas-reid — APPROVED** [review]

**Comment by zirain** [comment]
ping @istio/wg-user-experience-maintainers

**Review by stevenctl — APPROVED** [review]
cc @istio/wg-user-experience-maintainers @therealmitchconnors I'd like to get this into the release :)

**Comment by zirain** [comment]
ping @istio/wg-user-experience-maintainers

**Review by hanxiaop — APPROVED** [review]

**Pull request merged**